### PR TITLE
feat(bivariate-color-manager): 11535 - fix sorting of sublist items.

### PR DESCRIPTION
**Pull request by propakov**
Now we collect avgCorrelationY by yQuotientIndicator.name and use it for sorting sublist items. Before we just used avgCorrelationY for both X and Y sublist items.

before
image

now after fix
image

### CODE REVIEW

File: src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts [hunk 1]
```diff
@@ -227,3 +240,16 @@ export const bivariateColorManagerResourceAtom = createResourceAtom(
...
+  numeratorCorellationMap: NumeratorCorellationMap,
+): void => {
+  Object.values(bivariateColorManagerData).forEach((row) => {
+    [...Object.values(row.horizontal), ...Object.values(row.vertical)].forEach(
+      (layer) => {
+        layer.correlationLevel = numeratorCorellationMap[layer.name] || 0;
+      },
+    );
+  });
+};
```
Comment (Akiyamka) [inline src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts]: Just note:
Pretty unusual for me when function declared after they was used

**Review by Akiyamka — APPROVED** [review]

File: src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts (hunk 1 above)
Comment (propakov) [reply src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts]: this approach is from https://marekhudyma.com/code-style/2021/03/02/step-down-rule.html

so when you open file, key and high level content is on top, some util functions that usually about details(low level) are below

Comment (propakov) [reply src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts]: or sometimes I just unintentionally follow this rule
I'll put it on the top

Comment (Akiyamka) [reply src/features/bivariate_color_manager/atoms/bivariateColorManagerResource.ts]: pretty interesting rule

**Pull request merged**